src/commands/registration/stor.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     .tap(() => this.commandSocket.pause())
     .then(() => Promise.try(() => this.fs.write(fileName, {append, start: this.restByteCount})))
     .then((fsResponse) => {
-      let {stream, clientPath} = fsResponse;
+      let {stream, clientPath, pipeAllowed} = fsResponse;
```

Comment:
Is `pipeAllowed` meant to be an option? Currently `write` only returns stream and clientPath.

https://github.com/autovance/ftp-srv/blob/0b9167e1e44f41306925879a234f3d914a8b13f7/src/fs.js#L85-L88

Reply:
Technically a custom file system implementation could return this. But I'd rather avoid it at all costs.